SharpMap/Rendering/Symbolizer/RasterPointSymbolizer.cs

```diff
@@ -104,6 +116,41 @@ internal ImageAttributes ImageAttributes
             }
         }
 
+        /// <summary>
+        /// Construct imageattribute based upon Transparency and/or Color Re-map
+        /// </summary>
+        private void ConstructImgAttributes()
+        {
+            if (_imageAttributes != null)
+                _imageAttributes.Dispose();
+
+            if (SymbolColor == Color.Empty && Transparency <= 0)
+                return;
+
+            _imageAttributes = new ImageAttributes();
+
+            if (SymbolColor != Color.Empty)
+            {
+                var cm = new ColorMap[1];
+
+                var a = SymbolColor.A;
+
+                if (Transparency > 0)
+                    a = (byte)(Math.Ceiling(255 * (1F - Transparency)));
+
+                var nc = Color.FromArgb(a, SymbolColor);
+                cm[0] = new ColorMap();
+                cm[0].OldColor = Color.White;

```

Comment:
Why hard coded `System.Drawing.Color.White` instead of property `public Color RemapColor { get; set; }`

Reply:
Done